libs/service-portal/applications/src/screens/ApplicationList/ApplicationList.tsx

```diff
@@ -6,21 +6,29 @@ import {
 } from '@island.is/service-portal/core'
 import ApplicationCard from '../../components/ApplicationCard/ApplicationCard'
 import { Typography, Box, Stack } from '@island.is/island-ui/core'
-import { useListApplications } from '@island.is/service-portal/graphql'
+import { useApplicantApplications } from '@island.is/service-portal/graphql'
 import { Application } from '@island.is/application/core'
 
 const ApplicationList: ServicePortalModuleComponent = ({ userInfo }) => {
-  const { data: applications, loading, error } = useListApplications(
+  const { data: applications, loading, error } = useApplicantApplications(
     userInfo.profile.natreg,
   )
 
+  const {
+    data: assigneeApplications,
+    loading: assigneeApplicationsLoading,
+    error: assigneeApplicationsError,
+  } = useApplicantApplications(userInfo.profile.natreg)
```

Comment:
shouldnt you be using `useAssigneeApplications` here? Looks like you are using the applicant hook twice

Reply:
Yep, good catch!